src/product_exclusion.js

```diff
@@ -0,0 +1,3 @@
+const productExclusion = () => {}
```

Comment:
Is this going to be implemented in a further PR?

Reply:
The idea is to add exercises only but not the implementation. The master branch will contain only the exercise descriptions and some tests. 

So is not expected to have PRs for implementations